src/components/translator/i18n.ts

```diff
@@ -0,0 +1,27 @@
+import { initReactI18next } from 'react-i18next';
+import i18next from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+// Import your language files
+import ptTranslation from './pt.json';
+import frTranslation from './fr.json';
+import enTranslation from './en.json';
+
+// Define supported languages and their resources
+const supportedLanguages = ['pt', 'fr', 'en'];
+const resources = {
+  pt: { ohri: ptTranslation },
+  fr: { ohri: frTranslation },
+  en: { ohri: enTranslation },
```

Comment:
This key will have to be changed based off the previous comment about organization specific naming